geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/index/AttributeIdxStrategy.scala

```diff
@@ -335,6 +344,42 @@ object AttributeIdxStrategy extends StrategyProvider {
     }
   }
 
+  def attributeValueKvsToFeatures(sft: SimpleFeatureType,
+                                  indexSft: SimpleFeatureType,
+                                  returnSft: SimpleFeatureType,
+                                  attribute: String): FeatureFunction = {
+    import scala.collection.JavaConversions._
+    val attributeIndex = sft.indexOf(attribute)
+    val returnIndex = returnSft.indexOf(attribute)
+    val translateIndices =
+      indexSft.getAttributeDescriptors.map(d => returnSft.indexOf(d.getLocalName)).zipWithIndex.filter(_._1 != -1)
```

Comment:
yeah, since it's only done once I went foe brevity and simplicity over performance - think I should change it?